docs/docs/authentication/okta-sso.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Okta
+nav_order: 2
+parent: Authentication
+description: >
+  This page contains instructions on setting up Okta
+  as the SSO provider for Firezone.
+---
+---
+
+Firezone supports Single Sign-On (SSO) through Okta.
+After successfully configuring SSO with Firezone, users will be prompted to sign
+in with their Okta credentials to in the Firezone portal to authenticate VPN
+sessions, and download device configuration files.
+
+![Firezone Okta SSO Login](){:width="600"}
+
+To set up SSO, follow the steps below:
+
+## Step 1 - Create Okta App Integration
+
+_This section of the guide is based on
+[Okta's documentation](https://help.okta.com/en/prod/Content/Topics/Apps/Apps_Apps.htm)._
```

Comment:
Specifically, based on this:

https://developer.okta.com/docs/guides/sign-into-web-app-redirect/python/main/

Reply:
The section I linked is more written for the end user. This is the article that goes over how to integrate Okta with an app: https://help.okta.com/en/prod/Content/Topics/Apps/Apps_App_Integration_Wizard_OIDC.htm

Reply:
Makes sense.